src/player.h

```diff
@@ -718,6 +718,11 @@ class Player final : public Creature, public Cylinder
 				client->sendRemoveTileThing(pos, stackpos);
 			}
 		}
+		void sendRemoveTileCreature(const Creature* creature, const Position& pos, int32_t stackpos) {
+			if (client) {
```

Comment:
Can you please check if stackpos is -1 and not call client for those cases? I don't think the clients are designed to handle this -1.